kedro/framework/cli/utils.py

```diff
@@ -328,13 +331,18 @@ def load_entry_points(name: str) -> Sequence[click.MultiCommand]:
         List of entry point commands.
 
     """
-    entry_points = pkg_resources.iter_entry_points(group=ENTRY_POINT_GROUPS[name])
+    entry_points = importlib_metadata.entry_points()
+    entry_points = entry_points.select(group=ENTRY_POINT_GROUPS[name])
+
     entry_point_commands = []
     for entry_point in entry_points:
         try:
             entry_point_commands.append(entry_point.load())
-        except Exception as exc:
-            raise KedroCliError(f"Loading {name} commands from {entry_point}") from exc
+        except Exception as exc:  # pylint: disable=broad-except
+            logger.warning(
+                KedroCliError(f"Fail to load {name} commands from {entry_point}")
+            )
+            logger.warning(exc)
```

Comment:
This is the most important part. It's hard to determine from the framework side whether a kedro's run is still valid if plugins fails. `kedro-viz` is defintely ok to just ignore it, but for `kedro-mlflow` type of plugin that actually modify the run behavior,  it may be problematic so I am using `warning` level.

Reply:
Similar suggestion here to the other occurrence in cli.py.